### PR TITLE
Node Bootstrap Fix Ups

### PR DESCRIPTION
**Pull request by gambol99**
- following the suggestions made in the initial PR https://github.com/kubernetes/kops/pull/5253
- fixing up the variable name to following a more suitable convention
- fixing up the paths to use a strings.Join rather than a fiddly

### CODE REVIEW

File: nodeup/pkg/model/context.go [hunk 1]
```diff
@@ -313,8 +313,8 @@ func (c *NodeupModelContext) KubectlPath() string {
 
 // BuildCertificatePairTask creates the tasks to pull down the certificate and private key
 func (c *NodeupModelContext) BuildCertificatePairTask(ctx *fi.ModelBuilderContext, key, path, filename string) error {
-	certificateName := fmt.Sprintf("%s/%s.pem", strings.TrimSuffix(path, "/"), filename)
-	keyName := fmt.Sprintf("%s/%s-key.pem", strings.TrimSuffix(path, "/"), filename)
+	certificateName := strings.Join([]string{path, filename + ".pem"}, "/")
```
Comment (justinsb) [inline nodeup/pkg/model/context.go]: I actually meant `filepath.Join` https://golang.org/pkg/path/filepath/#Join but this works...

**Comment by justinsb** [comment]
/approve
/lgtm

Thanks @gambol99

File: nodeup/pkg/model/context.go (hunk 1 above)
Comment (gambol99) [reply nodeup/pkg/model/context.go]: aahh .. didn't know there was a `filepath.Join` .. actually that's better as does Clean() for us .. no need to Trim

**Pull request merged**